include/gz/physics/Joint.hh

```diff
@@ -584,6 +586,52 @@ namespace gz
       };
     };
 
+    /////////////////////////////////////////////////
+    /// \brief This feature sets the mimic constraint for this Joint.
+    class GZ_PHYSICS_VISIBLE SetMimicConstraintFeature
+        : public virtual Feature
+    {
+      /// \brief The Joint API for setting the mimic joint constraint.
+      /// \param[in] _dof
+      ///   The desired generalized coordinate corresponding to the follower
+      ///   axis within this joint. Values start from 0 and stop before
+      ///   Joint::GetDegreesOfFreedom().
+      /// \param[in] _joint
+      ///   name of the joint to be mimicked, i.e. the leader joint.
+      /// \param[in] _multiplier
+      ///   The multiplier to be applied to position of leader joint.
+      /// \param[in] _offset
+      ///   The offset to be applied to position of leader joint after
+      ///   the multiplier is applied.
+      public: template <typename PolicyT, typename FeaturesT>
+      class Joint : public virtual Feature::Joint<PolicyT, FeaturesT>
+      {
+        public: using Scalar = typename PolicyT::Scalar;
+
+        public: void SetMimicConstraint(
+                    const std::size_t _dof,
+                    const std::string _joint,
+                    const std::string _axis,
```

Comment:
use const references for `_joint` and `_axis`

Reply:
https://github.com/gazebosim/gz-physics/pull/431/commits/a7e4513a090b19089954edb24d07ce4a9a3083d3